code/datums/outputs.dm

```diff
@@ -4,7 +4,7 @@
 	var/text = ""
 	var/list/sounds = 'sound/items/airhorn.ogg' //can be either a sound path or a WEIGHTED list, put multiple for random selection between sounds
 	var/mutable_appearance/vfx = list('icons/sound_icon.dmi',"circle", HUD_LAYER) //syntax: icon, icon_state, layer
-	var/cooldown = 100
+	var/list/cooldown = list(100, FALSE) //1: how long is our cooldown, 2: are we cooling down or not. the 2nd value can be made into its own var but having 2 vars just for cooldown seems pretty dirty to me
```

Comment:
This makes datum outputs have universal cooldown for entire type, as opposed to having a separate cooldown for every instance of object using a datum output

Reply:
it literally doesn't straight up, look down

Reply:
this var just determines the period of the cooldown and checks if the period is over or not.